qase-testcafe/examples/.qaserc

```diff
@@ -1,7 +1,8 @@
 {
     "enabled": true,
     "logging": true,
-    "apiToken": "api_key",
-    "projectCode": "project_code",
+    "environmentId": 1,
```

Comment:
Let's change back "api_key" and "project_code". We used these "keywords" at Readme.md